result.go

```diff
@@ -381,17 +381,42 @@ func (q Query) ToMapBy(
 	res.Elem().Set(m)
 }
 
-// ToSlice iterates over a collection and populates the result slice with elements.
+// ToSlice iterates over a collection, copy to result slice the collection elements,
```

Comment:
@cleitonmarx no problem! None of us are native English speakers here. :) If you could check the box for "Allow edits from maintainers" on the right menu, I can help you with this commit.